### PR TITLE
mgr/volumes: return string type to ceph-manager

### PR DESCRIPTION
**Pull request by ajarr**
The 'volumes' ceph-mgr module returns a byte type instead of a string type as the stdout result of commands, `fs subvolume getpath` and `fs subvolumegroup getpath` to the ceph-mgr. This causes the commands to hang and the ceph-mgr to utilize 100% CPU. Fix the 'volumes' ceph-mgr module to return string type to ceph-mgr as the stdout result of the commands.

Fixes: https://tracker.ceph.com/issues/41903
Signed-off-by: Venky Shankar <vshankar@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by vshankar** [comment]
http://pulpito.ceph.com/vshankar-2019-09-18_15:10:12-fs-wip-vshankar-testing-distro-basic-ovh/

**Comment by vshankar** [comment]
@tchaikov, possible for you to take a look at this? This issue is hit w/ Python 3.

**Comment by vshankar** [comment]
> http://pulpito.ceph.com/vshankar-2019-09-18_15:10:12-fs-wip-vshankar-testing-distro-basic-ovh/

passed.

File: src/pybind/mgr/volumes/fs/volume.py [hunk 1]
```diff
@@ -419,7 +419,7 @@ def subvolume_getpath(self, fs_handle, **kwargs):
                 if not path:
                     raise VolumeException(
                         -errno.ENOENT, "Subvolume '{0}' not found".format(subvolname))
-                ret = 0, path, ""
+                ret = 0, path.decode("utf-8"), ""
```
Comment (tchaikov) [inline src/pybind/mgr/volumes/fs/volume.py]: see also 3d63cd947f55a8508f972ee249ea6a04836429ae

Comment (vshankar) [reply src/pybind/mgr/volumes/fs/volume.py]: yeh -- that's the commit that introduced this change. For this fix, we are only interested in calls that return to ceph-mgr DaemonServer.

**Pull request merged**